### ISSUE TITLE
Scene conditions

### ISSUE THREAD

**Issue by ferponcem** [individual-brain-charting/public_analysis_code#54]
When preparing the paper, the author of the **Scene** task suggested two things:
- calculating the behavioral scores independently for each condition, since it would be more informative due to the very different stimuli for each, see [table with expanded scores](https://github.com/individual-brain-charting/admin/blob/master/papers/data_paper5/supplementary_tables/scene_behavioral_allscores.ods) vs [table with all scores averaged as one](https://github.com/individual-brain-charting/admin/blob/master/papers/data_paper5/supplementary_tables/scene_behavioral.ods).
- changing the way we report the scores: instead of labeling the scores as `dot_easy` and `dot hard`, like we have in our documentation (see [Scene conditions](https://individual-brain-charting.github.io/docs/tasks.html#scene)), he suggests to use `possible_scrambled` and `impossible_scrambled`, to match the labels in their paper. 

I don't think it's an issue for the behavioral data we will send with the paper, however, I am wondering how should we publish this [behavioral data in our documentation](https://individual-brain-charting.github.io/docs/behavioral_data.html), if we should keep the original labels we've given to match the conditions list and file names or if we should publish them with the author's suggestions.

**Comment by bthirion** [comment]
Thx.
I propose to keep the author's suggestion for the paper. But we'll have to do some file renaming to run to match the changes required by the authors. This will be visible in the next release of the data on NeuroVault /EBrains.

**Comment by ferponcem** [comment]
Agreed! I will keep the issue open until the next release to make sure we take it into account. Thank you!!
